src/amber/router/scope.cr

```diff
@@ -1,9 +1,9 @@
 module Amber
   module Router
     class Scope
-      @stack = [] of String
+      property :stack
```

Comment:
Why leak the internal into the external world ?

No one should know `Scope` use an `Array`

Reply:
I change it back.